lib/client/api.go

```diff
@@ -1885,6 +1900,14 @@ func (tc *TeleportClient) SSH(ctx context.Context, command []string, runLocally
 	}
 	defer proxyClient.Close()
 
+	if tc.LoadAllHostCAs {
+		clusters, err := tc.LocalAgent().GetClusterNames()
+		if err != nil {
+			return trace.Wrap(err)
+		}
+		tc.LocalAgent().UpdateClusters(clusters...)
+	}
```

Comment:
`SSH()` method is a strange place to put this logic tbh. Does it cover all use-cases? E.g. what about scp? Why is this not a part of the `NewClient()`?

Reply:
I've moved this into `makeClientForProxy()`, which should make it work everywhere (scp, app and databases, etc.). It's only not in `NewClient()` because we have to do it in `makeClientForProxy()` for identity files anyway.